examples/beer-finder/src/NavBar.jsx

```diff
@@ -2,16 +2,16 @@ import React from 'react'
 import { view } from 'react-easy-state'
 import SearchBar from 'material-ui-search-bar'
 import LinearProgress from '@material-ui/core/LinearProgress'
-import appStore from './appStore'
+import { fetchBeers as handleOnRequestSearch, isLoading } from './appStore'
```

Comment:
This is kind of shady and probably breaks ):  appStore is exporting a single default object which is destructured here (these are not importing named exports). appStore is basically like a MobX store so destructuring here will likely not result in the view re-rendering on `isLoading` changes (depending on how babel treats this stuff). Did standard style force you to do this?

Reply:
Yeah, I can see the problem here. Standard only forced me to rename fetchBeers, I thought It would be best to just create an alias.